src/main/java/figures/Color.java

```diff
@@ -0,0 +1,17 @@
+package figures;
+
+public enum Color {
+    GREEN,
+    RED,
+    BLUE,
+    BLACK,
+    WHITE,
+    YELLOW,
+    PURPLE;
+
+    public static Color randomColor() {
```

Comment:
```suggestion
    public static Color getRandomColor() {
```
methods naming is usually a verb
https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names